src/App.tsx

```diff
@@ -1,26 +1,202 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+/*eslint-disable*/
+import { useEffect, useMemo, useState } from 'react';
+
+import {
+  USER_ID,
+  getTodos,
+  createTodo,
+  deleteTodo,
+  updateToDo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { ToDoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error/Error';
+import React from 'react';
+
 
-const USER_ID = 0;
+
+
+function getVisibleToDos(newTodos: Todo[], newStatus: Status) {
+  switch (newStatus) {
+    case Status.Active:
+      return newTodos.filter(todo => !todo.completed);
+
+    case Status.Completed:
+      return newTodos.filter(todo => todo.completed);
+
+    default:
+      return newTodos;
+  }
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [status, setStatus] = useState(Status.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [title, setTitle] = useState('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isAllLoading, setIsAllLoading] = useState<boolean>(false);
```

Comment:
the loadings should be set to true by default in most cases, but I think your logic is kinda different